src/Enterspeed.Source.Sdk/Domain/Services/EnterspeedIngestService.cs

```diff
@@ -187,13 +133,25 @@ public Response Ingest<T>(IEnterspeedEntity<T> entity, IEnterspeedConnection con
             string responseContentAsString = null;
             try
             {
-                var content = _jsonSerializer.Serialize(entity);
+                string jsonEntityToIngest;
+                string ingestUrl;
+                if (ingestVersion == "1")
+                {
+                    jsonEntityToIngest = _jsonSerializer.Serialize(entity);
+                    ingestUrl = _ingestEndpoint;
```

Comment:
_ingestEndpoint and _ingestEndpointV2 are kinda funky, as _ingestEndpoint also has the version baked in. Also why is the version a string?

Reply:
@jthind Committed refactoring. https://github.com/enterspeedhq/enterspeed-sdk-dotnet/pull/31/commits/4469b5ba6b356f5912c6319f922fc369a3ed0e98